docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: 'Set the default Implementation'
+sidebar_position: 1
+---
+
+# Set the default Implementation
+
+:::note
+
+Users deploying their Universal Profiles using the guides that utilize **[lsp-factory](../universal-profile/create-profile.md)** or the **[Browser Extension](../browser-extension/create-a-universal-profile.md)** can skip this guide, as this contract is already deployed and set for their profiles.
+
+:::
+
+This guide will teach you how to deploy and set the default implementation of the **[Universal Receiver Delegate](../../standards/smart-contracts/lsp1-universal-receiver-delegate-up.md)** (URD) used by the Universal Profile. This contract will register the addresses of the **[received assets](../../standards/universal-profile/lsp5-received-assets.md)** and **[vaults](../../standards/universal-profile/lsp10-received-vaults.md)** and will remove them on a balance equal to 0. This contract requires the [SETDATA Permission](../../standards/universal-profile/lsp6-key-manager.md#permissions) to interact with the profile through the KeyManager.
+
+![UniversalReceiverDelegate setting data keys on profile](/img/token-transfer-4.jpg)
+
+## Deploy the default Universal Receiver Delegate contract
+
+```typescript title="Deploying the Universal Receiver Delegate of the Universal Profile"
+import Web3 from 'web3';
+import LSP1UniversalReceiverDelegateUP from '@lukso/lsp-smart-contracts/artifacts/LSP1UniversalReceiverDelegateUP.json';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+
+const PRIVATE_KEY = '0x...'; // your EOA private key
+const myEOA = web3.eth.accounts.wallet.add(PRIVATE_KEY);
+
+// create an instance
+let myURD = new web3.eth.Contract(LSP1UniversalReceiverDelegateUP.abi);
+
+// deploy the URD contract
+await myURD
+  .deploy({
+    data: LSP1UniversalReceiverDelegateUP.bytecode
+  })
+  .send({
+    from: myEOA.address,
+    gas: 5_000_000,
+    gasPrice: '1000000000',
+  });
+```
+
+### Set the address of the URD in the storage
+
+After deploying the contract, we will need to set its address under the **[LSP1-UniversalReceiverDelegate Data Key](../../standards/generic-standards/lsp1-universal-receiver.md#extension)** and grant it the SETDATA permission.
```

Comment:
```suggestion
After deploying the contract, we will need to set its address under the **[LSP1-UniversalReceiverDelegate Data Key](../../standards/generic-standards/lsp1-universal-receiver.md#extension)** and grant it the `SETDATA` permission.
```

Reply:
Well done, this will help lots of dev in understanding the URD's functionalities!